src/pages/index.js

```diff
@@ -26,7 +26,7 @@ function FrameMask(props){
   return(
     <div className={props.className} onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
     <iframe src={props.src} width={props.width} height= {props.height} frameborder="0" scrolling="no"></iframe>
-    <div style={maskStyle}  hidden={!mouseState.state}  onClick={()=>{window.open(props.src)}}>
+    <div style={maskStyle}  hidden={!mouseState.state}>
```

Comment:
remove `onClick`. If not removed, one click will open two pages. like: [casdoor.org](https://casdoor.org/)

Reply:
good!